pkg/checkpoint/crutils/checkpoint_restore_utils.go

```diff
@@ -75,7 +75,7 @@ func CRImportCheckpointConfigOnly(destination, input string) error {
 // it exists deletes all files listed.
 func CRRemoveDeletedFiles(id, baseDirectory, containerRootDirectory string) error {
 	deletedFiles, _, err := metadata.ReadContainerCheckpointDeletedFiles(baseDirectory)
-	if os.IsNotExist(errors.Unwrap(errors.Unwrap(err))) {
+	if os.IsNotExist(errors.Unwrap(err)) {
```

Comment:
this should just be `errors.Is(err, os.ErrNotExist)` there is no need to manually unwrap.

Reply:
removed the unwrapping